doc/changes.txt

```diff
@@ -2,6 +2,15 @@
 
 @page changes Release Notes
 
+@htmlonly <a name="v7_0_1_changes"></a>@endhtmlonly
+@par
+<B>Version 7.0.1</B> - <I>In Development</I>
+
+@par
+Bug fixes:
+- Fixed a bug where empty grids were still returning true from
```

Comment:
This description was confusing to me since an empty grid (as defined by the Grid::empty() = Tree::empty()) is a Tree without any nodes. However, this method computes the bounding box of active values (irrespective of where they live in the tree). May I therefore suggest this wording instead:

- Fixed a bug where grids with no active values might return true when the method evalActiveVoxelBoundingBox is called. The correct behavior is to only return true if the grid contains any active values.